packages/backend/src/scripts/configure-env.ts

```diff
@@ -0,0 +1,91 @@
+import * as fs from 'fs';
+import * as dotenv from 'dotenv';
+
+const PATH_DEV = './.env.development';
+const PATH_PROD = './.env.production';
+
+/**
+ * configure the environment for .env.development and .env.production if not existing
+ * @param  {} filePath - The path of environment files in backend
+ */
+function validateEnvironmentFile(filePath) {
+  console.log(`Checking the existence of ${filePath}`);
+
+  try {
+    fs.accessSync(filePath, fs.constants.F_OK);
+    console.log(`${filePath} exists`);
+    return validateKeys(filePath);
+  } catch (error) {
+    console.error(
+      `An unexpected error ${error} is encountered \n'${filePath}' file does not exist, creating from '${filePath}.template '`,
+    );
+
+    // copy the data from template file and paste to wanted files
+    const data = fs.readFileSync(`${filePath}.template`, 'utf-8');
+
+    fs.writeFileSync(filePath, data);
+
+    console.log(
+      `'${filePath}' file created successfully \n Please start it again.`,
+    );
+    return false;
+  }
+}
+
+/**
+ * Check the environment contains all keys from its template files
+ * @param  {} filePath - The path of environment files in backend
+ */
+function validateKeys(filePath) {
+  console.log(`Starting to validate ${filePath}`);
+
+  // Checks the existence of the environment template file
+  fs.accessSync(`${filePath}.template`, fs.constants.F_OK);
+
+  // Read the data from the environment file and its template
+  const data = fs.readFileSync(`${filePath}`, 'utf-8');
+  const dataTemplate = fs.readFileSync(`${filePath}.template`, 'utf-8');
+
+  // Parse the keys
+  const environmentConfig = Buffer.from(data);
+  const templateConfig = Buffer.from(dataTemplate);
+
+  const environmentObject = dotenv.parse(environmentConfig);
+  const templateObject = dotenv.parse(templateConfig);
+
+  const environmentKeys = Object.keys(environmentObject);
+  const templateKeys = Object.keys(templateObject);
+
+  console.log(
+    `Key comparsions result - environment : [${environmentKeys}] template : [${templateKeys}]`,
+  );
```

Comment:
I'd prefer if this only printed when there was a difference between the two key sets but it's just a log so it's fine.